ports/stm32/boards/NUCLEO_H563ZI/pins.csv

```diff
@@ -0,0 +1,130 @@
+,PA0
+,PA1
+,PA2
+,PA3
+,PA4
+,PA5
+,PA7
+,PA8
+,PA9
+,PA10
+,PA11
+,PA12
+,PA13
+,PA14
+,PA15
+,PB0
+,PB2
+,PB3
+,PB4
+,PB5
+,PB6
+,PB7
+,PB8
+,PB9
+,PB10
+,PB11
+,PB12
+,PB13
+,PB14
+,PB15
+,PC1
+,PC4
+,PC5
+,PC6
+,PC7
+,PC8
+,PC9
+,PC10
+,PC11
+,PC12
+,PC13
+,PC14
+,PC15
+,PD0
+,PD1
+,PD2
+,PD3
+,PD4
+,PD5
+,PD6
+,PD7
+,PD8
+,PD9
+,PD10
+,PD11
+,PD12
+,PD13
+,PD14
+,PE0
+,PE1
+,PE2
+,PE3
+,PE4
+,PE5
+,PE6
+,PE7
+,PE8
+,PE10
+,PE12
+,PE15
+,PF0
+,PF1
+,PF2
+,PF4
+,PF5
+,PF6
+,PF7
+,PF8
+,PF9
+,PF10
+,PF12
+,PF13
+,PF14
+,PF15
+,PG0
+,PG1
+,PG2
+,PG3
+,PG4
+,PG5
+,PG6
+,PG7
+,PG8
+,PG9
+,PG10
+,PG11
+,PG13
+,PG15
+_A0,PA6
```

Comment:
These shouldn't start with an underscore.  Other Nucleo boards use `A0` etc.

Reply:
Can you please help me with the definitions in pins.csv. From my understanding the entries create "aliases" which can be used when instantiating a pin.

Why did I add the underscore? In my tests I found that i.e. naming PA6 as (Arduino) A0, shadows the PA0 pin of the STM32. 

Is there a (Micro)Pythonic solution to this problem?

Reply:
I've actually put together a tool to auto generate the stm pins CSV file if you want to try it out? 

https://github.com/micropython/micropython/pull/12272

Reply:
@andrewleech Thanks for bringing this to my attention. I'll give it a try. Please note that the pin css file for STM32H5 was already created. I am mainly creating the board support for the NUCLEO-H563.

Reply:
Yeah sorry I realised later I'd mixed up chip af CSV and board pin CSV in my mind.

Reply:
@renestraub Yes I understand there is a name clash with Arduino names and CPU names.

We do need to try and improve that situation.  For the time being it's possible to disambiguate by using `Pin.board.A0` vs `Pin.cpu.A0`.

Please do remove the leading underscores, so this board matches all the other boards that have defined the Arduino pins.

Then in the future we can try to fix this properly for all boards.

Reply:
Done

Reply:
Thanks.